### DISCUSSION TITLE
Monaco Editor as code editor plugin

### DISCUSSION THREAD

**Discussion by ZzzzzzzSkyward** [Milkdown/milkdown#195] [General]
How to implant a code editor in milkdown, given an instance from `monaco.create()` and a `div` element to begin with?
ProseMirror shows an example, but I cannot follow that example, being confused by Remark plugin, where I'm eager to find some `dom` attribute to load the editor instead of some text or array.

**Comment by Saul-Mirone** [comment]
The remark plugin will help you to convert markdown to prosemirror node, but how to render a prosemirror node is defined by node schema and view.
